.github/workflows/release.yml

```diff
@@ -44,6 +44,17 @@ jobs:
 
       - name: Run storybook build
         run: make storybook
+
+    - name: Commit storybook changes
+      uses: EndBug/add-and-commit@v4
+      with:
+        author_name: Max Strübing
+        author_email: max.struebing@farbenmeer.de
+        message: "docs: Update storybook"
```

Comment:
To prevent github actions to run again add the `skip ci` part
```suggestion
        message: "docs: Update storybook [skip ci]"
```

Reply:
Could you explain what this does exactly and why we need this?

Reply:
wrong asumption. If you push something from an github action it will not trigger another github action. So its fine as it is.

Otherwise there would be a loop of commit docs -> start github action -> commit docs -> start github cations ....

Reply:
No its not, you have to add 2 spaces on each line, 
```yml
    - name: Commit storybook changes
```
is not on the same line with 
```yml
      - name: Run storybook build
```

See here:
![Bildschirmfoto 2020-06-23 um 18 57 55](https://user-images.githubusercontent.com/14888529/85432723-a746ce00-b583-11ea-9bca-d8e7b208c4f3.png)